.github/workflows/build_test.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         - name: setup php
           uses: shivammathur/setup-php@v2
           with:
-            php-version: '8.1'
+            php-version: '8.2'
```

Comment:
Can we put both 8.1 and 8.2?

Reply:
This is for code coverage only so I think run it with one version would be enough. We don't have much PHP version related implementation.